components/vf-card-container/README.md

```diff
@@ -26,7 +27,7 @@ The containier defaults to three columns (the recommended number of image-based
 
 The `vf-card` component allows for a CSS custom property to be set to define the `aspect-ratio` of the card image. This can be set per card, but with the `vf-card-container` you should aim for consistency across the `vf-card`s it is displaying. Therefore we can pass the `aspect-ratio` value from this component and it will cascade through the CSS onto the cards. This is encapsulated to this container, which allows you to set different `aspect-ratio` values for different containers as needed.
 
-To set the `aspect-ratio` you will need to set the `card_custom_aspect_ratio` key/value pair in the `.yml` or the `&#x7B;&#x25;&#x20; render &#x20;&#x25;&#x7D;` api of the `vf-card-container as needed.
+To set the `aspect-ratio` you will need to set the `card_custom_aspect_ratio` key/value pair in the `.yml` or the `{% render %}` api of the `vf-card-container` as needed.
```

Comment:
@nikiforosk can you please add same syntax here `{#{% include %}#}`

Reply:
@sandykadam I think we discussed to release the vf-card change in next sprint or no?  I thought it'll require a vf-card changelog so we decided to do it in next sprint I guess.

Reply:
Yes this is vf-card-container change not vf-card. I commented above because it will again fail the build if Niki don't add comment to code, changelog he has done but same is there in readme.md